latent.ipynb

```diff
@@ -988,8 +989,10 @@
         "            mask = Image.open(fetch(init_mask)).convert('RGB')\n",
         "            mask = TF.to_tensor(mask).to(device).unsqueeze(0)\n",
         "            if mask_rotate: mask = torch.rot90(mask, 1, [3,2])\n",
-        "            mask = F.interpolate(mask,[opt.H//8,opt.W//8]).mean(1)\n",
+        "            #mask = F.interpolate(mask,[opt.H//8,opt.W//8]).mean(1)\n",
+        "            mask = resize(mask,out_shape = [opt.n_samples,4,opt.H//8, opt.W//8])\n",
         "            mask = transform(mask)\n",
+        "            mask = torch.round(mask)\n",
```

Comment:
i intentionally left the mask as float. In this implementation, the float value controls the propotion of steps a pixel to be masked.
like, if a pixel value is 0.5, only the first 50% steps is masked. After that it will enter normal diffusion process.